packages/expo-checkbox/src/ExpoCheckbox.web.tsx

```diff
@@ -42,7 +42,7 @@ const ExpoCheckbox: CheckboxComponent = (props) => {
   );
 
   const nativeControl = createElement('input', {
-    accessibilityState: { disabled, checked: value },
+    accessibilitystate: { disabled, checked: value },
```

Comment:
👀

Reply:
i was told by react that this was needed but let me double check this

Reply:
<img width="1465" alt="image" src="https://user-images.githubusercontent.com/90494/199105263-62cda677-84c4-4d49-95b3-67010fd3cb93.png">

😰

Reply:
Cool, thanks for double-checking 😉

Reply:
proper fix: https://github.com/expo/expo/pull/19791/commits/9c1276c3e0583fe8ef635b3c951f923ed11a5976